.github/workflows/outside-pr.yml

```diff
@@ -18,35 +22,57 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: recursive
-      - name: Get all git tags
-        run: git fetch --prune --unshallow --tags
-      - name: Setup Go for Building
+          fetch-depth: 0
+
+      - name: Setup Go
         uses: actions/setup-go@v2
         with:
-          go-version: '^1.17.2'
+          go-version: ${{ env.SETUP_GO_VERSION }}
+
       - name: Setup Ginkgo Test Framework
-        run: go get -u github.com/onsi/ginkgo/v2/ginkgo
+        run: go install github.com/onsi/ginkgo/v2/ginkgo@v2.1.1
+
       - name: Cache Tools
         uses: actions/cache@v2
         with:
           path: ${{ github.workspace }}/tools
           key: ${{ runner.os }}-tools
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
Being logged it to dockerhub was supposed to be optional. If you are not logged in you are subject to rate limiting but we thought it would be ok for the few external PRs. At some point we broke CI and it didn't succeed without dockerhub credentials.

In any case, external PRs don't have access to our secrets. I don't know how external contributors should define these then (define the secrets on their fork?). With us having moved away from dockerhub for many of our images, maybe we are fine to skip login all together. In the main CI pipeline, I think we even use a mirror registry like we do locally.

You can leave it as it is and handle dockerhub login on another PR if you prefer.

Reply:
> I don't know how external contributors should define these then (define the secrets on their fork?).
Yes, they should be defined in their fork, it will work in that case.

> You can leave it as it is and handle dockerhub login on another PR if you prefer.
Maybe better yes, as it may have more impacts.

Reply:
>>You can leave it as it is and handle dockerhub login on another PR if you prefer.
>
>Maybe better yes, as it may have more impacts.

I created an issue for this: https://github.com/epinio/epinio/issues/1229